docs/dev/libvirt-howto.md

```diff
@@ -339,7 +339,7 @@ Error: Error applying plan:
 Depending on your libvirt version you might encounter [a race condition][bugzilla_libvirt_race] leading to an error similar to:
 
 ```
-* libvirt_domain.master.0: Error creating libvirt domain: virError(Code=43, Domain=19, Message='Network not found: no network with matching name 'tectonic'')
+* libvirt_domain.master.0: Error creating libvirt domain: virError(Code=43, Domain=19, Message='Network not found: no network with matching name 'openshift'')
```

Comment:
I think we create network with `cluster-name` name

Reply:
You are correct. I've changed it to `test1`.